frontend/src/App.js

```diff
@@ -1,15 +1,35 @@
 
 import './App.css';
-import Map from "./components/Map";
-import Overview from "./components/Overview";
+import OverviewPage from "./pages/OverviewPage";
+import AppHeader from "./components/AppHeader";
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom'
+import React from "react";
+import NewRoutePage from "./pages/NewRoutePage";
+import RoutesOverview from "./components/RoutesOverview";
+import RouteDetails from "./pages/RouteDetails";
 
 
 function App() {
   return (
-    <div>
-      <Map/>
-      <Overview/>
-    </div>
+    <Router>
+        <Switch>
+            <Route exact path="/">
+                <AppHeader/>
+                <div>
+                    <OverviewPage/>
+                </div>
+            </Route>
+            <Route path="/newroutepage">
+                <NewRoutePage/>
+            </Route>
```

Comment:
```suggestion
            <Route path="/newroutepage" component={NewRoutePage}/>
```
What do you think about this? short and sweet :)

Reply:
Oh yeah ! looks much better !